src/agent/plugins/build/prepare.ts

```diff
@@ -133,6 +124,8 @@ export function beforeJob(executionContext: cm.IExecutionContext, callback) {
         return scmProvider.getCode();
     })
     .then((code: number) => {
```

Comment:
No because if the scm provider fails, it goes to the .fail handler.  We're just printing out the return code as information.  Should most likely be 0 since we're in the success block.